modules/Authentication.php

```diff
@@ -1,11 +1,17 @@
 <?php
 	class Authentication {
-		private $user;
+		public $user;
 
-		function __construct($email, $password) {
-			if (!($email && $password)) {
-				return false;
+		public function __construct($email, $password) {
+			if (!$email || !$password) {
+				return NULL;
```

Comment:
Instead of returning null in a constructor, throw an exception. This is sort of a standard behavior.